site/content/en/docs/tasks/dev/integrate_a_custom_job.md

```diff
@@ -6,63 +6,99 @@ description: >
   Integrate a custom Job with Kueue.
 ---
 
-Kueue integrates with a couple of Jobs, including Kubernetes batch Job, MPIJob, RayJob or JobSet.  
+Kueue has built-in integrations for several Job types, including
+Kubernetes batch Job, MPIJob, RayJob and JobSet.
 
-There are two options for integrating a Job-like CRD with Kueue:
-- As part of the Kueue repository
+There are two options for adding an additional integration for a Job-like CRD with Kueue:
 - Writing an external controller
+- As part of the Kueue repository
 
-This guide is for [platform developers](/docs/tasks#platform-developer) and focuses on the first approach.
-If there is a widely used Job which you would like supported in Kueue, please consider contributing.
-This page shows how to integrate a custom Job with Kueue.
-
-## Pre-requisites
-
-1. The custom Job CRD should have a suspend-like field, with semantics similar to the [`suspend` field in a Kubernetes Job](https://kubernetes.io/docs/concepts/workloads/controllers/job/#suspending-a-job).
+This guide is for [platform developers](/docs/tasks#platform-developer) and describes how
+to build a new integration.  We recommend building your new integration as an external
```

Comment:
Same as above. I'm not sure if the external integration is the recommended option in all of the cases. For private, company-specific CRDs - definitely yes and it is the only possibility. But for public, well-adopted CRDs the decision is, imho, not that clear at this stage of Kueue maturity. 

@alculquicondor @tenzen-y please chime in.

Reply:
I revised as requested.

As a practical matter, building an integration as an experimental integration isn't that attractive.  If the downstream project can't enable their controller without providing a bespoke built of Kueue, it isn't that useful.